README.md

```diff
@@ -31,79 +30,121 @@ A current version can be viewed in Git Markdown from this repository:
+You will need [Python Poetry](https://python-poetry.org/docs/) to run the
+SSP Toolkit in a Python virtual environment. Once you have Poetry
+installed you will be able to run all the commands using the
+format `poetry run [COMMAND]`.
 
-```bash
-docker --version
-docker-compose --version
-```
 
-### Activate your environment
+## Generating the documentation
+
+To update the local Markdown and or to create new exportable files, perform the following steps:
+
+### createfiles
+
+Create/update the frontmatter, components and appendices using [templates](templates) and [keys](keys)
 
-We use the [bowline](https://github.com/CivicActions/bowline/) docker sandbox helper to instantiate local containers with [compliancetools](https://github.com/CivicActions/compliancetools) and [secrender](https://github.com/CivicActions/secrender). The following two commands will ***activate*** your local environment, setting up local aliases for the bowline-exposed commands which are described below.
+#### Example
 
 ```bash
-docker-compose pull
-source activate
+poetry run createfiles -i configuration.yaml -t templates
 ```
 
-## Generating the documentation
+#### Usage
 
-To update the local Markdown and or to create new exportable files, perform the following steps:
+```bash
+Usage: createfiles [OPTIONS]
 
-1. Create/update the frontmatter, components and appendices using [templates](templates) and [keys](keys)
+Options:
+  -t, --templates DIRECTORY  Template directory
+  -o, --out PATH             Output directory (default: current directory)
+  --help                     Show this message and exit.
+```
 
+### makefamilies
+
+Generate markdown versions of the RMF control implementation family files in the `docs/controls/` directory:
+
+#### Example
 ```bash
-createfiles -i configuration.yaml -t templates
+poetry run makefamilies
 ```
 
-2. Generate markdown versions of the RMF control implementation family files in the `/docs/controls/` directory:
+### sop
+
+Generate Standard Operating Procedure (SOP) docs (from `components/` and `keys/sop.yaml`) in the `docs/sop` directory
 
+#### Example
 ```bash
-mkdir -p docs/controls
-makefamilies
+poetry run sop -c components
 ```
 
-3. Generate Standard Operating Procedure (SOP) docs (from `components/` and `keys/sop.yaml`) in the `docs/sop` directory
+#### Usage
+```bash
+Usage: sop [OPTIONS]
+
+Options:
+  -c, --components DIRECTORY  Rendered components directory
+  -o, --out PATH              Output directory (default: docs/)
+  --help                      Show this message and exit.
+
+```
 
-    ```bash
-    sop -i configuration.yaml -c components -o docs
-    ```
+### makessp
 
-4. Generate Microsoft Word (.docx) versions of the control family files (see the `docx/` directory):
+Generate System Security Plan (SSP)
 
+#### Example
 ```bash
-exportto -c docs/controls
+poetry run makessp
 ```
 
-5. Generate Microsoft Word (.docx) versions of the appendices and front matter (also in `docx/` directory):
+### exportto
```

Comment:
One warning:
```
$ poetry run exportto -r appendices/
Writing file docx/sdlc.docx
Writing file docx/risk-management.docx
Writing file docx/configuration-management.docx
Writing file docx/security-irp.docx
Writing file docx/contingency-plan.docx
Writing file docx/privacy-impact-assessment.docx
[WARNING] Could not fetch resource 'None': PandocResourceNotFound "None"

Writing file docx/system-continuity-plan.docx
Writing file docx/security-irp-checklist.docx
Writing file docx/laws-regulations.docx
Writing file docx/justifications.docx
```

Reply:
There was an image missing from  the keys files.